server/src/pages/shotindex/view.js

```diff
@@ -81,7 +84,7 @@ class Body extends React.Component {
     } else {
```

Comment:
nit: we can move masonryOptions definition from line 63 in else block where its used.

Reply:
Good point. Fixed.